src/odemis/gui/cont/settings.py

```diff
@@ -308,7 +308,7 @@ def adjust_done(_):
         def auto_adjust(_):
             """ Call the auto contrast method on the detector if it's not already running """
             if not btn_autoadjust.up:
-                f = detector.applyAutoContrast()
+                f = detector.applyAutoContrast() # TODO renaming required here?
```

Comment:
yes

Reply:
Do I have to make changes here? Theses files i.e. delphi.py and dist-packages are read-only. Due to which I do not have editing rights.
![image](https://user-images.githubusercontent.com/37955981/194048411-3d39f8d8-4575-4155-95f6-19aeb8d44612.png)